### ISSUE TITLE
Move an item to the top of contextmenu

### ISSUE THREAD

**Issue by iG8R** [averissimo/open-link-in-foreground#1]
Hi. Could you please make it possible to move "Open link in foreground" to the top of contextmenu like it done in 
https://addons.mozilla.org/en-US/firefox/addon/fullscreenelement/
https://addons.cdn.mozilla.net/user-media/previews/full/234/234795.png
And - "yes", I know I could do it through userchrome.css:
```
#open_link_in_foreground_yahoo_com-menuitem-_open-link-foreground { 
    -moz-box-ordinal-group: 0 !important;/**/
}
```
but for some unknown reason after such manipulation with userchrome.css all other add-on's items was moved to the top.

**Comment by GerdchenF** [comment]
> but for some unknown reason after such manipulation with userchrome.css all other add-on's items was moved to the top.

How do you mean that? All items cannot be at the top together.
It works for me, but I have not used the !important (also don't want to try because never change a running system ;-) )

**Comment by averissimo** [comment]
I don't think that the menu option should be on top of the default "Open Link in a New Tab", maybe it should be the second item on the menu
